src/NzbDrone.Core.Test/BulkImport/AddMultiMoviesFixture.cs

```diff
@@ -5,12 +5,11 @@
 using NzbDrone.Core.Organizer;
 using NzbDrone.Core.Movies;
 using NzbDrone.Core.Test.Framework;
-using NzbDrone.Core.Movies.Events;
 using System.Collections.Generic;
 
 namespace NzbDrone.Core.Test.BulkImport
 {
-	[TestFixture]
+    [TestFixture]
 	public class AddMultiMoviesFixture : CoreTest<MovieService>
```

Comment:
Formatting of this file needs to be fixed. The lines you touch seem to be good, but rest is in tabs.